backend/src/api/endpoints/authentication.py

```diff
@@ -109,3 +129,22 @@ def refresh_access_token(
         token_type="bearer",
         refresh_token_expires=new_token.expires_at,
     )
+
+
+@router.get(
+    "/content",
+    summary="Returns success if the user can access the content",
+)
+async def auth_content(
+    request: Request,
+    db: Session = Depends(get_db_session),
+    x_original_uri: Annotated[str | None, Header()] = None,
+):
```

Comment:
Mini suggestion here. If you have time, please add ` -> None` to indicate that the function returns nothing on purpose :-)

Reply:
I actually expected a bool and only fully understood it when I was reading the frontend part which basically says everything except an error is fine ^^